percona-xtradb-cluster-5.7/Dockerfile

```diff
@@ -42,24 +41,24 @@ RUN set -ex; \
     percona-release enable pxc-57 ${PXC_REPO}
 
 RUN set -ex; \
-    curl -Lf -o /tmp/numactl-libs.rpm http://vault.centos.org/centos/8/BaseOS/x86_64/os/Packages/numactl-libs-2.0.12-13.el8.x86_64.rpm; \
-    curl -Lf -o /tmp/libev.rpm http://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/libev-4.24-6.el8.x86_64.rpm; \
-    curl -Lf -o /tmp/jq.rpm http://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/jq-1.5-12.el8.x86_64.rpm; \
-    curl -Lf -o /tmp/oniguruma.rpm http://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/oniguruma-6.8.2-2.el8.x86_64.rpm; \
-    curl -Lf -o /tmp/pv.rpm http://download.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/p/pv-1.6.6-7.el8.x86_64.rpm; \
-    curl -Lf -o /tmp/socat.rpm http://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/socat-1.7.4.1-1.el8.x86_64.rpm; \
+    curl -O https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-9; \
+    rpm --import RPM-GPG-KEY-EPEL-9; \
+    curl -Lf -o /tmp/numactl-libs.rpm https://downloads.percona.com/downloads/packaging/numactl-libs-2.0.18-2.el9.x86_64.rpm; \
+    curl -Lf -o /tmp/libev.rpm https://downloads.percona.com/downloads/packaging/libev-4.33-5.el9.x86_64.rpm; \
+    curl -Lf -o /tmp/jq.rpm https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/jq-1.6-14.el9.x86_64.rpm; \
+    curl -Lf -o /tmp/oniguruma.rpm https://downloads.percona.com/downloads/packaging/oniguruma-6.9.6-1.el9.6.x86_64.rpm; \
+    curl -Lf -o /tmp/pv.rpm http://download.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/p/pv-1.6.20-1.el9.x86_64.rpm; \
+    curl -Lf -o /tmp/socat.rpm https://downloads.percona.com/downloads/packaging/socat-1.7.4.1-6.el9.x86_64.rpm; \
```

Comment:
Do we have any reason to install socat from Percona repo? As In can see UBI9 has the same version `socat;1.7.4.1-6.el9;x86_64;ubi-9-appstream-rpms ` the same about `oniguruma`.

Reply:
@hors Thanks for pointing this out. Fixed the issue.

Reply:
@hors Can you please check now?